WalletWasabi/Wallets/WalletManager.cs

```diff
@@ -59,15 +60,15 @@ public WalletManager(Network network, string workDir, WalletDirectories walletDi
 	private object Lock { get; } = new();
 	private AsyncLock StartStopWalletLock { get; } = new();
 
-	private BitcoinStore BitcoinStore { get; set; }
+	private BitcoinStore? BitcoinStore { get; set; }
```

Comment:
We agreed before to not do this, just by adding a ?. All of these warnings mean that the code might be incorrect and we should try to dismiss nullability - Lucas

Did you do the analysis if the code is correct?